lib/openapi/mongoid/spec_builder.rb

```diff
@@ -409,6 +417,14 @@ def build_crud_specification(routes)
                       key :'$ref', name
                     end
                   end
+
+                  response 422 do
+                    key :description, 'UnprocessableEntityError'
```

Comment:
Yes, description supposed to be free form text, in other places we use those to show operationId in the document.